core/src/main/java/com/scalar/db/transaction/jdbc/JdbcTransactionManager.java

```diff
@@ -140,6 +140,11 @@ public TransactionState getState(String txId) {
     return TransactionState.UNKNOWN; // always returns UNKNOWN
   }
 
+  @Override
+  public TransactionState abort(String txId) {
+    return TransactionState.UNKNOWN; // always returns UNKNOWN
```

Comment:
I rethought this method and `getState` method.
I now think it's better to throw an exception since the application cannot rely on the APIs.
(I think, If UNKNOWN is returned, users would expect if it is returning something else sometimes or eventually)
Thought?

Reply:
Thank you for reviewing this!

Yes, I agree that the methods should throw an exception (`UnsupportedOperationException`) in the JDBC transaction. I will fix it in that way. Thanks!